rust-arroyo/src/processing/dlq.rs

```diff
@@ -435,13 +435,21 @@ impl<TPayload> BufferedMessages<TPayload> {
         }
 
         buffered.push_back(message.clone());
+        Self::report_buffered_metrics(partition_index, buffered);
+    }
 
-        // Number of elements that can be held in buffer deque without reallocating
+    fn report_buffered_metrics<T>(partition_index: u16, buffered: &VecDeque<T>) {
         gauge!(
             "arroyo.consumer.dlq_buffer.capacity",
             buffered.capacity() as u64,
             "partition_id" => partition_index
         );
+
+        gauge!(
+            "arroyo.consumer.dlq_buffer.len",
+            buffered.capacity() as u64,
```

Comment:
Is this a typo? both are reporting `.capacity()`

Reply:
yes, thank you :D